lib/src/github/models.rs

```diff
@@ -96,6 +96,40 @@ pub struct App {
     pub name: String,
 }
 
+#[derive(Deserialize, Serialize, Clone, Debug)]
+pub struct Team {
+    pub name: String,
+    pub slug: String,
+    pub html_url: String,
+}
+
+impl Team {
+    pub fn new(org: &str, slug: &str) -> Team {
+        Team {
+            name: slug.to_string(),
+            slug: slug.to_string(),
+            html_url: format!(
+                "https://github.com/orgs/{}/teams/{}",
+                org.to_string(),
+                slug.to_string()
+            ),
+        }
+    }
+
+    // org is not an explicit field on a team object nor in any other types. So we are going to
+    // parse it from the `html_url` field.
+    pub fn org(&self) -> &str {
+        // TODO: grab org from html_url
+        "eng"
```

Comment:
hmm. looks like there is an `org` passed into the constructor. let's just store that as a member as well? or do you mean when de-serializing from json, there is no `org` field?

Reply:
Correct, there is no org field when deserializing from json.

Reply:
just had a thought. teams i think always exist in the same org as the PR, right? or, is it like `org/team-name`? probably can parse it from there instead of parsing it from the html url. i'll look at same examples.

Reply:
Yes, they exist in the same org. `slug` field on a Team does not seem to include the org 😞